l1-contracts/contracts/upgrades/UpgradeHyperchains.sol

```diff
@@ -0,0 +1,33 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity 0.8.24;
+
+import {Diamond} from "../state-transition/libraries/Diamond.sol";
+import {BaseZkSyncUpgrade, ProposedUpgrade} from "./BaseZkSyncUpgrade.sol";
+import {ETH_TOKEN_ADDRESS} from "../common/Config.sol";
+
+/// @author Matter Labs
+/// @custom:security-contact security@matterlabs.dev
```

Comment:
Could you please add a comment that this is the upgrade that will be used to migrate era to a be a part of the shared bridge/bridgehub ecosystem